frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -130,13 +122,11 @@ pods:
           size: {{DATA_NODE_DISK}}
         cmd: |
           set -e
-
```

Comment:
Similar comment as above regarding the whitespace.

Reply:
done, restored the white space.